<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSettingsDialog</class>
 <widget class="QDialog" name="QSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>350</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>320</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Session settings</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>300</width>
     <height>301</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="documentMode">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="WidgetCascade">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QGroupBox" name="groupBoxCascade">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>10</y>
       <width>271</width>
       <height>81</height>
      </rect>
     </property>
     <property name="title">
      <string>Path to a cascade file</string>
     </property>
     <widget class="QLineEdit" name="lineEditCascade">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>haarcascades/haarcascade_frontalface_alt.xml</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ButtonCascade">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>30</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBoxCascade">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Use cascade classifier for region detection</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBoxTimer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Time between successive frequency evaluations, ms</string>
     </property>
     <widget class="QSlider" name="horizontalSliderTimer">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>20</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksBothSides</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditTimer">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MicraC</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>1000</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBoxVideoFile">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Enroll video file instead of webcam</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="WidgetEvaluation">
    <attribute name="title">
     <string>Advanced</string>
    </attribute>
    <widget class="QGroupBox" name="groupBoxDatalength">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>10</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Length of data</string>
     </property>
     <widget class="QDial" name="dialDatalength">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="minimum">
       <number>5</number>
      </property>
      <property name="maximum">
       <number>11</number>
      </property>
      <property name="sliderPosition">
       <number>8</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditDatalength">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MicraC</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>256</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBoxBufferlength">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>271</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Length of buffer</string>
     </property>
     <widget class="QDial" name="dialBufferlength">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>60</width>
        <height>60</height>
       </rect>
      </property>
      <property name="minimum">
       <number>4</number>
      </property>
      <property name="maximum">
       <number>8</number>
      </property>
      <property name="value">
       <number>8</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEditBufferlength">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MicraC</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>256</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="clearButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBoxFFT">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>if unchecked, frequency will be evaluated from digital filter output</string>
     </property>
     <property name="text">
      <string>Use harmonic analysis for frequency estimation</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>271</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabPatient">
    <attribute name="title">
     <string>Patient</string>
    </attribute>
    <widget class="QGroupBox" name="groupBoxPatient">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>10</y>
       <width>271</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>Population data</string>
     </property>
     <widget class="QLineEdit" name="lineEditPatient">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>45</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>normal_heart_rate_at_rest.xml</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonPatient">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>45</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBoxPatient">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>From 25 % to 75 %</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>From 10 % to 90%</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>From 5 % to 95 %</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>From 2,5 % to 97,5 %</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>From 1,0 % to 99,0 %</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Path to file with distribution</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Select interval:</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEditPatient">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
       </font>
      </property>
      <property name="date">
       <date>
        <year>1988</year>
        <month>8</month>
        <day>23</day>
       </date>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBoxPatient">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Use population data</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>117</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Select birth date:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonMale">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>181</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Male</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonFemale">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>181</y>
       <width>61</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Female</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>180</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Select patient sex:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="ButtonCancel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Choose to cancel processing</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonAccept">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Choose to start processing</string>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonDefault">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>320</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Choose to set default values</string>
   </property>
   <property name="text">
    <string>Set default</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
